### ISSUE TITLE
ARIA role for mwc-ripple

### ISSUE THREAD

**Issue by steverep** [material-components/material-web#2986]
The `mwc-ripple` custom element produces unnecessary nodes in the accessibility tree (one for the custom element and one for the child `div` in the shadow DOM).  Since this component is entirely just meant to be a CSS insertion hook for animation, the element and its child should be given the [ARIA none role](https://www.w3.org/TR/wai-aria-1.1/#none), i.e. add the attribute `role="none presentation"`.

**Comment by dfreedm** [comment]
Makes sense, I'll try to play with that soon and see how it goes.

**Comment by steverep** [comment]
@dfreedm  can you take a look at PR #3087 to close this?

**Comment by dfreedm** [comment]
Yeah sorry, I meant to comment on that yesterday.

I haven't been able to replicate VoiceOver or NVDA announcing anything for the ripple element via object navigation, so I'm pretty sure this change isn't necessary.

**Comment by MrHBS** [comment]
@dfreedm I think the point is to remove it from the tree altogether so you can't even navigate to it.

**Comment by steverep** [comment]
> I haven't been able to replicate VoiceOver or NVDA announcing anything for the ripple element via object navigation, so I'm pretty sure this change isn't necessary.

So, I was going to comment on how to do it, but now I cannot navigate to it either on any of the demo components I've checked.  That was definitely not the case when I filed this, so something must have changed in the past few months either with browsers or OS.

It used to be that after landing on a `mwc-icon-button`, for example, NVDA could navigate all the children, including the empty ripple `div`.  Oddly enough, the ripple element is still in the accessibility tree (see attached).  I don't have a tree from the past to compare, but it looks like it is now being labelled as "offscreen" which I don't remember being the case.

[mwc-icon-button-accessibility-tree.zip](https://github.com/material-components/material-web/files/8291616/mwc-icon-button-accessibility-tree.zip)

In any case, `role="none"` or `aria-hidden="true"` is still the correct and safest treatment here because it's an element that marks up nothing and exists solely to inject CSS.  You could use any block element and get the same effect - the semantic role of the element means nothing.  From the [ARIA 1.2 spec list of example use cases](https://www.w3.org/TR/wai-aria-1.2/#note-regarding-the-aria-1-1-none-role-0):

> An element used as an additional markup "hook" for CSS; or
